pandas/tests/indexes/test_numeric.py

```diff
@@ -26,6 +26,36 @@ def full_like(array, value):
     return ret
 
 
+class TestIndexArithmetic(object):
+    @pytest.mark.parametrize('index', [pd.Int64Index(range(1, 11)),
+                                       pd.UInt64Index(range(1, 11)),
+                                       pd.Float64Index(range(1, 11)),
```

Comment:
is this how we are importing in this module, e.g. ``pd.*``? usually we don't do this (its ok whichever, just want to be consistent)